shared/src/packages/surveys/libs/validation-schemas/create-survey.validation-schema.ts

```diff
@@ -25,6 +25,47 @@ const createSurveyValidationSchema = joi.object<SurveyRequestDto, true>({
     .messages({
       'array.min': SurveyValidationMessage.MINIMUM_PREFERENCE_LENGTH,
     }),
+  feelings: joi
```

Comment:
let's combine these validation schemas instead of defining one more time

Reply:
done

Reply:
Not resolved

Reply:
Fix this comment

Reply:
I unify all!

Reply:
done!

Reply:
still not fixed, don't write that this is done until changes are in the branch

Reply:
Sorry I forget delete this file